src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -609,29 +608,10 @@ public Value first() {
      * or the <tt>defaultValue</tt> otherwise
      */
     @SuppressWarnings("unchecked")
-    @Nonnull
-    public <V> V get(Class<V> clazz, @Nonnull V defaultValue) {
-        Object result = get(defaultValue);
-        if (!clazz.isAssignableFrom(result.getClass())) {
-            return defaultValue;
-        }
-        return (V) result;
-    }
-
-    /**
-     * Returns the wrapped value if it is an instance of the given clazz or <tt>null</tt> otherwise.
-     *
-     * @param clazz the desired class of the return type
-     * @param <V>   the expected type of the wrapped value
-     * @return the wrapped value if the given <tt>clazz</tt> is assignable from wrapped values class or <tt>null</tt>
-     * otherwise
-     */
-    @Nullable
-    @SuppressWarnings("unchecked")
-    public <V> V get(Class<V> clazz) {
+    public <V> V get(Class<V> clazz, V defaultValue) {
```

Comment:
couldnt we retain the get(Class) method with just an overload of (Class, null) to not break as much code? my guess is that a lot of these calls could also be in templates where now breaking code is not as easy to detect.
And it wouldnt increase the complexity a lot